pubspec.yaml

```diff
@@ -12,60 +12,56 @@ analyzer:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_bloc: ^6.1.2
-  equatable: ^1.2.6
-  url_launcher: ^5.7.10
+  flutter_bloc: ^7.0.0
+  equatable: ^2.0.0
+  url_launcher: ^6.0.3
   step_progress_indicator: ^0.2.5+8
-  intl: ^0.16.0
+  intl: ^0.17.0
   flutter_localizations:
     sdk: flutter
-  i18n_extension: ^1.3.7
-  cupertino_icons: ^0.1.3
-  http: ^0.12.0+4
-  shared_preferences: ^0.5.6+1
-  flutter_secure_storage: ^3.3.2
-  local_auth: ^0.6.2+3
-  timeago: ^2.0.26
-  cached_network_image: ^2.0.0
-  intro_views_flutter:
-    git: git://github.com/n13/IntroViews-Flutter
+  i18n_extension: ^4.0.0
```

Comment:
our dependencies had not been updated in a long long time, we need to do a better job of keeping them up to date
@RaulUrtecho  @n13